src/app/modules/main/wallet_section/filter.nim

```diff
@@ -24,9 +25,19 @@ proc `$`*(self: Filter): string =
     )"""
 
 
+proc setFillterAllAddresses*(self: Filter) = 
+  self.addresses = self.controller.getWalletAccounts().map(a => a.address)
+
+proc toggleWatchOnlyAccounts*(self: Filter) =
+  self.excludeWatchOnly = not self.excludeWatchOnly
```

Comment:
@alaibe maybe you have an idea how to do it, but just want to share some thoughts on this. 

A request is that user should be able to include/exclude watch only accounts in total balance from the settings and as well from the wallet section. Also that's the setting which need to be stored and that maintained per user, not per the app installation.

Now based on that fact I don't see filter as a good place for maintaining `excludeWatchOnly` value.

It should be stored in let's say new table `accounts_settings` (where we will store settings per account (having columns `key_uid`, `entity`, `string_value`, `int_value`, `bool_value`, defining `key_uid` as foreign key which  references `accounts.key_uid`)).

Then in wallet service we need to have a function `toggleWatchOnlyAccountsInBalances` which will toggle the value in db first and then send signal `INCLUDE_WO_ACC_IN_BALANCE_CHANGED` sending true or false as payload. 

Then we need to connect to that signal in wallet section and wallet settings. When we receive that signal in wallet section, from there we need to update filter calling `self.filter.includingWatchOnlyAccountsInBalancesUpdated(arg.value)`.

Reply:
I merge this one and work on persist it